### ISSUE TITLE
Consider styling the loyalty costs of planeswalker loyalty abilities.

### ISSUE THREAD

**Issue by NilsEnevoldsen** [scryfall/feedback#39]
Just as you use mana symbols for the costs of activated abilities like “B:”, so you should use loyalty symbols for the costs of activated abilities like “+1:”.

**Comment by Sniffnoy** [comment]
Worth noting that Gatherer doesn't do this; personally that makes me a bit skeptical it's a good idea, since I'd think we would want display of Oracle text to match.

**Comment by NilsEnevoldsen** [comment]
I think it's just a limitation of Gatherer. I believe the loyalty symbol is as much a part of the "text" as a mana or tap symbol.

> “An activated ability with a loyalty symbol in its cost is a loyalty ability.”

**Comment by silasary** [comment]
mtg.wtf has recently started experimenting with [doing this](http://mtg.wtf/card?q=Jace%20Beleran).

I'm not sure whether I like it or not.

**Comment by NilsEnevoldsen** [comment]
I like it.

**Comment by Thom1729** [comment]
I agree that planeswalker text looks better with the formatted icons. I think that the same could could be done for leveler cards and sagas.

**Comment by NilsEnevoldsen** [comment]
That's true; all three are defined symbols in the comprehensive rules. For reference, loyalty symbols are 107.7, level symbols are 107.8, and chapter symbols are 107.15.
